.travis.yml

```diff
@@ -0,0 +1,58 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - "8"
+
+stages:
+  - lint
+  - deploy
+
+jobs:
+  include:
+    - stage: lint
+      name: EditorConfig Syntax Verification
+      install:
+        - npm install -g eclint
+        - eclint --version
+      script: eclint check
+    - stage: lint
+      name: Shell Script Syntax Verification
+      script: make sh/lint
+    - stage: lint
+      name: JSON Lint/Format Verification
+      script: make json/lint
+    - stage: lint
+      name: Terraform Lint/Format Verification
+      install:
+        - make terraform/install
+        - make terraform-docs/install
+      script:
+        - make terraform/lint
+        - make docs/lint
```

Comment:
Since _this project_ isn't actually a tf module, can remove this and `make terraform-docs/install` (or adjust the make docs/lint target to just lint markdown)

Reply:
Cleaned this up from the travis and Makefile